src/components/KCodeBlock/KCodeBlock.vue

```diff
@@ -480,21 +509,20 @@ watch(() => props.highlightedLineNumbers, function() {
   setDefaultMatchingLineNumbers()
 }, { immediate: true, deep: true })
 
-watch(() => isShowingFilteredCode.value, async function() {
+watch(() => isShowingFilteredCode.value, function(value) {
```

Comment:
nit: since this is a ref, you don't need the getter. Also, can we rename the param something besides `value` (there's enough "value" in Vue already 😅 )
```suggestion
watch(isShowingFilteredCode, function(isFilteredCode) {
```